src/engine/renderer/tr_image.cpp

```diff
@@ -808,8 +808,7 @@ level 1 has only numLayers/2 layers. There are still numLayers pointers in
 the dataArray for every mip level, the unneeded elements at the end aren't used.
 ===============
 */
-void R_UploadImage( const byte **dataArray, int numLayers, int numMips,
-		    image_t *image )
+void R_UploadImage( const byte **dataArray, int numLayers, int numMips, image_t *image, imageParams_t *imageParams )
```

Comment:
A new argument is added but not used anywhere?

Reply:
Hmmm, probably a leftover of all those will rebases…

Reply:
After all those rebase, this change belonged to the next commit, so I moved it to the next commit.